packages/app/components/NoContentInbox.tsx

```diff
@@ -0,0 +1,22 @@
+import { Paragraph, Button, Heading, YStack, Spacer } from '@internal/ui'
+import { useRouter } from 'solito/router'
+
+export default function NoContentInbox() {
+  const { back } = useRouter()
+
+  return (
+    <YStack jc="space-between" px="$4" height="95%">
+      <Spacer />
+      <YStack>
+        <YStack jc="center" ai="center" gap="$2">
+          <Heading variant="h2">You're all caught up.</Heading>
+          <Paragraph textAlign="center" secondary>
+            You don't have any notifications at the moment.
```

Comment:
Maybe this a bit smaller than the title?

Reply:
I kept it the same as the home screen when you don't have any crednetials yet. Shouild i also change that you think for consistency?

Reply:
Merged for now, but let me know, and I'll make a follow up PR